File: Dart/LinearSearch.dart
int linearSearch(List<int> arr, int target) {
  for (int i = 0; i < arr.length; i++) {
    if (arr[i] == target) {
      return i; // Return the index of the target element if found
    }
  }
  return -1; // Return -1 if the target element is not found in the list
}

void main() {
  List<int> arr = [5, 2, 9, 1, 5, 6];
  int target = 9;

  int result = linearSearch(arr, target);

  if (result != -1) {
    print('Element $target found at index $result');
  } else {
    print('Element $target not found in the list');
  }
}

File: JS/LinearSearch.js
function linearSearch(arr, target) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === target) {
      return i; // Return the index of the target element if found
    }
  }
  return -1; // Return -1 if the target element is not found in the array
}

// Example usage:
const arr = [3, 7, 1, 9, 4, 2];
const target = 9;
const result = linearSearch(arr, target);

if (result !== -1) {
  console.log(`Element ${target} found at index ${result}`);
} else {
  console.log(`Element ${target} not found in the array`);
}
